core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -376,7 +376,7 @@ object Boot extends IOApp with LazyLogging {
         shardedExecutionServiceCluster,
         conf.getInt("executionservice.batchSize"),
         workspaceManagerDAO,
-        dataRepoDAO,
+        deltaLayer,
```

Comment:
the `dataRepoDAO` was unused in WorkspaceService, so I just replaced it with the deltaLayer object.